packages/react/src/tooltip/Tooltip.js

```diff
@@ -74,103 +83,172 @@ const Tooltip = forwardRef((
         willRemove: true,
       });
     }, (hideDelay !== undefined));
-
-    useOnceWhen(() => {
-      warnRemovedProps('shouldWrapChildren', {
-        prefix,
-        message: 'Use Function as Child Component (FaCC) to render the tooltip trigger instead.',
-      });
-    }, (shouldWrapChildren !== undefined && !shouldWrapChildren));
   }
 
   const anchorRef = useRef(null);
   const nodeRef = useRef(null);
   const combinedRef = useForkRef(anchorRef, ref);
   const isHydrated = useHydrated();
+  const canDisplayTooltip = (
+    !disabled &&
+    !isBlankString(label) && // '', ' ', '\t\r\n'
+    !isEmptyArray(label) && // []
+    !isEmptyObject(label) && // {}
+    !isNullOrUndefined(label) // null or undefined
+  );
```

Comment:
When component could still work, I would like to just check if label is assigned without too many validation.
`!disabled && !isBlankString && !!label`
or 
`!disabled && !!label`
not to stop user if they really want to show tooltip with spaces  XD

Reply:
Thank you for raising this question.

`!!value` is the most simple way to coerce the value to be either `true` or `false`. Although it will also treat numeric zero (`0`) as a falsy value, it can be mitigated using a zero string (`'0'`).

For this PR, I will go for `!disabled && !isBlankString && !!label` because leading and trailing whitespace chaacters are ignored in HTML, as well as the `title` attribute.